layouts/page/nx-cugraph.html

```diff
@@ -0,0 +1,325 @@
+{{- define "main" -}}
+
+<!-- Hero Partial -->
+{{ partial "hero.html" (dict
+"text" "GPU Accelerated NetworkX Backends"
+"btn_text" "Try Now"
+"btn_link" "https://github.com/rapidsai/nx-cugraph"
```

Comment:
Maybe this should match cudf-pandas?
```suggestion
"btn_text" "Get Started"
"btn_link" "#get-started"
```